vm-virtio/src/iommu.rs

```diff
@@ -189,11 +221,11 @@ unsafe impl ByteValued for VirtioIommuReqUnmap {}
 
 /// Virtio IOMMU request PROBE types
 #[allow(unused)]
-const VIRTIO_IOMMU_PROBE_T_MASK: u32 = 0xfff;
+const VIRTIO_IOMMU_PROBE_T_MASK: u16 = 0xfff;
 #[allow(unused)]
-const VIRTIO_IOMMU_PROBE_T_NONE: u32 = 0;
+const VIRTIO_IOMMU_PROBE_T_NONE: u16 = 0;
 #[allow(unused)]
```

Comment:
I guess we could remove this now...?

Reply:
Yes that's a very good point, I removed all the unnecessary `unused` from this file.